src/core/tsi/ssl_transport_security.c

```diff
@@ -49,6 +49,8 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+#include "src/core/support/env.h"
```

Comment:
Re:ciphersuites. Do you mean the use of an environment variable here?
https://github.com/grpc/grpc/blob/master/src/core/security/security_connector.c#L74